ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -105,6 +105,27 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
           />
         }
       />
+      <OptionsEditorControl
+        label={'Palette'}
+        control={
+          <ToggleButtonGroup
```

Comment:
Added palette visual editing quickly so we can switch back and forth between 'Categorical' and 'Auto' palettes, this UX is going to change drastically and the hard-coded ToggleButtonGroup was quickest (Mohit seemed okay with it now, but he prefers that it becomes a dropdown later) 

I can also spend a bit more time on this if people think it is worth it now?